source/common/http/conn_pool_grid.h

```diff
@@ -91,6 +91,8 @@ class ConnectivityGrid : public ConnectionPool::Instance,
                                   const StreamInfo::StreamInfo& info,
                                   absl::optional<Http::Protocol> protocol);
 
+    bool http3_attempt_failed() const { return http3_attempt_failed_; }
```

Comment:
You and your google habits.  no snake case for you. :-P

http3AttemptFailed()
ditto for brokenness below

Reply:
Done! I renamed is_http3_broken() and set_is_http3_broken(). I *think* that's all?

Reply:
Done! I renamed is_http3_broken() and set_is_http3_broken(). I think that's all?